lib/components/molecules/Form/DDFileInput/DDFileInput.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/FileInput';
+import styles from './DDFileInput.style';
+import useOpenFileDialog from '../../../../core/utils/useOpenFileDialog';
+
+type Props = {
+  id: string,
+  className?: string,
+  type?: string,
+  name: string,
+  label: Node,
+  labelProps?: Object,
+  fieldProps?: Object,
+  value?: string | number,
+  placeholder?: string,
+  onChange?: Function,
+  onBlur?: Function,
+};
+
+const { FileInputField } = useOpenFileDialog();
```

Comment:
The way you have named useOpenFileDialog is a Hook but that not how we use hooks. You are using it like utility functions even those should allow you to import/export individual functions

Reply:
Changed the implementation, other than returning a object inside object, returning individual properties and methods like hooks.